RecoBTag/TensorFlow/python/pfDeepFlavour_cff.py

```diff
@@ -3,11 +3,14 @@
 from RecoBTag.FeatureTools.pfDeepFlavourTagInfos_cfi import pfDeepFlavourTagInfos
 from RecoBTag.FeatureTools.pfNegativeDeepFlavourTagInfos_cfi import pfNegativeDeepFlavourTagInfos
 from RecoBTag.FeatureTools.pfDeepDoubleBTagInfos_cfi import pfDeepDoubleBTagInfos
+from RecoBTag.FeatureTools.pfDeepDoubleCvLTagInfos_cfi import pfDeepDoubleCvLTagInfos
+from RecoBTag.FeatureTools.pfDeepDoubleCvBTagInfos_cfi import pfDeepDoubleCvBTagInfos
 
 from RecoBTag.TensorFlow.pfDeepFlavourJetTags_cfi import pfDeepFlavourJetTags
 from RecoBTag.TensorFlow.pfNegativeDeepFlavourJetTags_cfi import pfNegativeDeepFlavourJetTags
 from RecoBTag.TensorFlow.pfDeepDoubleBJetTags_cfi import pfDeepDoubleBJetTags
-
+from RecoBTag.TensorFlow.pfDeepDoubleCvLJetTags_cfi import pfDeepDoubleCvLJetTags
+from RecoBTag.TensorFlow.pfDeepDoubleCvBJetTags_cfi import pfDeepDoubleCvBJetTags
 from CommonTools.PileupAlgos.Puppi_cff import puppi
 from PhysicsTools.PatAlgos.slimming.primaryVertexAssociation_cfi import primaryVertexAssociation
 
```

Comment:
shouldn't the pfDeepDoubleBTagInfos, pfDeepDoubleCvLTagInfos etc be added to pfDeepFlavourTask below?
Otherwise, what's the point of making the edits in this file to import the modules?

Reply:
Seems to be necessary, otherwise values are not filled in test